profile/Repository names

```diff
@@ -0,0 +1,19 @@
+blog: Home to the Blog for Public Code and where blog posts are planned and collectively written.
+
+standard: The Standard for Public Code - a model for public organizations to build their own open source solutions to enable successful future reuse by similar public organizations in other places.
+
+jitsi-community: A tool that allows users to sign up to a Jitsi instance that is not open to everyone, using a web UI so that they can start sessions by themselves.
```

Comment:
```suggestion
jitsi-community: A tool that allows community members to host and run video calls on our self-hosted Jitsi instance. This tool enables users to sign up, create rooms, and start calls as a moderator.
```

Reply:
No, this tool could be used on any self-hosted Jitsi instance, not only ours. But if the "our" is changed to "a" it is good.

Reply:
```suggestion
jitsi-community: A tool that allows users to sign up, create and run video calls on a self-hosted Jitsi instance. We use this to allow our community members to run calls on our infrastructure.
```

Aha! Got it. Does this fit better?